src/index.html

```diff
@@ -15,8 +15,42 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="logo"></div>
+    <div class="lista">
+      <ul>
+        <li>
+          <a href="#">
+            <img
+              src="./images/logo.png"
+              alt="imagem da logo"
+            />
+          </a>
+        </li>
+        <li><a href="#">APPLE</a></li>
+        <li><a href="#">SAMSUNG</a></li>
+        <li><a href="#">SMARTPHONES</a></li>
+        <li data-qa="hover"><a href="#">LAPTOPS & COMPUTERS</a></li>
+        <li><a href="#">GADGETS</a></li>
+        <li><a href="#">TABLETS</a></li>
+        <li><a href="#">PHOTO</a></li>
+        <li><a href="#"></a></li>
+      </ul>
+    </div>
+    <div class="is-active"></div>
```

Comment:
This `div` is not needed. You need to add the is-active class on a `a` tag on list. Like `<a href="#" class="is-active">APPLE</a></li>`